fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -56,8 +56,9 @@
 
 public class TableFunctionTable extends Table {
 
-    private static final String PROPERTY_PATH = "path";
-    private static final String PROPERTY_FORMAT = "format";
+    public static final String FAKE_PATH = "fake://";
```

Comment:
It's better to add clear commit message for big pull request.
The background, the solution and the syntax.

Reply:
good advice